opg-nginx-1604/docker/etc/nginx/nginx.conf

```diff
@@ -64,6 +64,7 @@ http {
                              '"query_string": "$query_string", '
                              '"uri": "$uri", '
                              '"request_path": "$request_path" '
+                             '"zf_matched_route_name": "$http_x_sirius_zf_matched_route_name", '
```

Comment:
Have you tested what the logs look like when you don't send the header through?
Anything appearing in nginx error log?

Reply:
confirmed. it defaults to blank string.